### DISCUSSION TITLE
How to override default paths like app_config_dir

### DISCUSSION THREAD

**Discussion by LeoMartinDev** [tauri-apps/tauri#10545] [General]
I have a binary that I duplicate to generate standalone applications on the flight.
These apps are portable, so values returned by `BaseDirectory` like `BaseDirectory.AppConfig` are related to the build of the binary which is now irrelevant in my use case

**Comment by FabianLars** [comment]
You can't really do that since those paths are basically OS apis (plus the bundle-id for the AppX methods). If only the bundle id is the issue you can use the normal base dirs and append your own ids to that. If the base paths are the problem then you simply won't be able to use them.

**Reply by LeoMartinDev** [reply]
thanks
